native_store.go

```diff
@@ -120,6 +122,9 @@ func (ns *nativeStore) Put(ctx context.Context, serverAddress string, cred auth.
 		return err
 	}
 	_, err = ns.exec.Execute(ctx, bytes.NewReader(credJSON), "store")
+	if err != nil && err.Error() == errExeNotInPathMessage {
+		return fmt.Errorf("credentials store is configured to `desktop` but Docker Desktop seems not running")
```

Comment:
nit:
```suggestion
		return fmt.Errorf("credentials store is configured to `desktop.exe` but Docker Desktop seems not running")
```
/cc @FeynmanZhou

Reply:
@shizhMSFT Good catch.

Reply:
I think `desktop.exe` is too Windows-specific, and `desktop` is a better choice. Currently the problem only appears in WSL and it is caused by a recent regression. There's no guarantee that future regressions will not cause similar issues on other OSs.